network_simulator/statistics.py

```diff
@@ -16,8 +16,8 @@ class LinkStats(object):
 
     :ivar list buffer_occupancy: a list of (timestamp, buffer_occupancy)
     tuples.
-    :ivar list packet_loss_times: a list of timestamps, each corresponding
-    to a packet loss.
+    :ivar list packet_loss_times: a list of (timestamps, packet_size), each
```

Comment:
Isn't this just a list of timestamps? I think we just count number of packet losses, not the size of them.